src/components/common/Stepper.vue

```diff
@@ -1,53 +1,51 @@
 <template>
   <v-card id="step-buttons-container">
-    <template v-for="(step, index) in getSteps">
-      <div
-        class="step"
-        :key="index"
```

Comment:
eslint complained about the key not being in the template, but if I changed that then eslint complained about a key in a template.

So, I combined the template and the div tags, and eslint is happy.

Reply:
PS If this diff looks weird in GH, click the ⚙️ icon at the top of this page and enable the Hide Whitespace option.